docs/topics/whatsnew/whatsnew22.md

```diff
@@ -247,12 +247,12 @@ In the example below, the `@Email` annotation is applied to all relevant targets
 data class User(
     val username: String,
 
-    // Applies @Email to param, property, field,
-    // get, and set_param (if var)
+    // Applies @Email to `param`, `property`, `field`,
+    // `get`, and `setparam` (if var)
     @all:Email val email: String,
 ) {
-    // Applies @Email to property, field, and getter 
-    // (no param since it's not in the constructor)
+    // Applies @Email to `property`, `field`, and `get`
+    // (no `param` since it's not in the constructor)
```

Comment:
```suggestion
    // Applies @Email to property, field, and get
    // (no param since it's not in the constructor)
```

Reply:
We don't use backticks in code comments. I fixed this in the original doc.

Reply:
I merged your changes from `master` branch